datarobot_batch_scoring/main.py

```diff
@@ -252,12 +254,18 @@ def main(argv=sys.argv[1:]):
     skip_row_id = parsed_args['skip_row_id']
     output_delimiter = parsed_args.get('output_delimiter')
 
+    if pid and not lid:
```

Comment:
@mckaymatt @Axik 
I thought on conversion it to the option type. But then it'll break the notation of supplying model & project id as arguments. Also, pid & lid arguments should become optional as well. And it will change batch_scoring CLI interface completely.